atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -231,8 +273,8 @@ def run(self):
                                         '", "'.join(sorted(unknown_states)))
 
         rebuild = is_rebuild(self.workflow)
-        success = not self.workflow.build_result.is_failed()
-        canceled = self.workflow.autorebuild_canceled
+        success = not self.workflow.build_process_failed
+        canceled = self.workflow.autorebuild_canceled or self.workflow.build_canceled
```

Comment:
I'm not sure if it is - and it hard to try it out without an integration test. 

Lets keep it and adjust it later on when chained rebuilds are testable